src/components/Chat/InputMessageV2.tsx

```diff
@@ -170,6 +372,11 @@ const InputMessageV2 = ({
     );
   };
 
+  const plusButtonStyle = React.useCallback(() => {
+    if (type === SIGNED) return colors.darkBlue;
+    return colors.bondi_blue;
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `plusButtonStyle` function has been introduced to determine the color of the plus button based on the chat type. This seems like a reasonable addition, but make sure that the colors returned by this function are valid and match the desired design.



<!-- This is an auto-generated comment by OSS CodeRabbit -->